{{cookiecutter.project_name}}/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-12-31
```

Comment:
probably safer to have a date that remains obviously wrong for the foreseeable future

```suggestion
## [0.0.1] - 1900-12-31
```